client/me/purchases/payment-methods/payment-method-details.tsx

```diff
@@ -48,12 +48,7 @@ const PaymentMethodDetails: FunctionComponent< Props > = ( {
 			/>
 			<div className="payment-method-details__details">
 				<span className="payment-method-details__number">
-					{ getPaymentMethodSummary( {
-						translate,
-						type,
-						digits: lastDigits,
-						email,
-					} ) }
+					{ <PaymentMethodSummary type={ type } digits={ lastDigits } email={ email } /> }
```

Comment:
Minor: you can remove the curly braces around this now that it's pure JSX.

Reply:
Removed.